Tasks/Buhayev/Task6/db.py

```diff
@@ -0,0 +1,134 @@
+import pandas as pd
+import sys
+from datetime import date
+
+class Base():
```

Comment:
actually not the best usage of OOP, considering this class is required to work with a one specific type of css 'db'

I would suggest to do all these stuff without a class or to make this class a "static"-like by using '@staticmethod' or '@classmethod' for all the behavior